app/main.py

```diff
@@ -109,12 +106,23 @@ async def knowledge_panel(
     sec_value_tag: Optional[str] = QueryData.secondary_value_tag_query(),
     lang_code: Optional[str] = QueryData.language_code_query(),
     country: Optional[str] = QueryData.country_query(),
+    include: Annotated[list[PanelName] | None, QueryData.include_panel_query()] = None,
+    exclude: Annotated[list[PanelName] | None, QueryData.exclude_panel_query()] = None,
 ):
     """
     FacetName is the model that have list of values
     facet_tag are the list of values connecting to FacetName
     eg:- category/beer, here beer is the value
     """
+    if include is not None and exclude is not None:
+        raise HTTPException(status_code=400, detail="include and exclude parameters are exclusive")
+
+    panel_names = {PanelName.hunger_game, PanelName.wikidata}
```

Comment:
Those are the defaults ?

In this case we should document it.

I was thinking panel_names would have them all, and we would change off-server code to only ask the one we want. 

But it's ok for a transition period.

Reply:
As other panels are expensive for the server, I think it's better to have sane default. I'm adding it to the documentation.